Dockerfile.centos7

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
+
+RUN yum update -y && yum install -y centos-release-ansible-27 centos-release-scl-rh && \
+    yum -y install ansible && \
+    yum -y install epel-release && \
```

Comment:
I guess you can install this in the first step and add ansible to `$PKGS` to simplify the command line.

Reply:
https://github.com/linux-system-roles/test-harness/pull/87/commits/a1f7e36ec876eb927bb6ddcd25ba619b13dc88c0